cost/azure/databricks/rightsize_compute/README.md

```diff
@@ -0,0 +1,126 @@
+# Azure Databricks Rightsize Compute Instances
+
+## What it does
+
+This policy checks all the instances associated with Azure Databricks workspaces in Azure Subscriptions for the average or maximum CPU and/or memory usage over a user-specified number of days. If the usage is less than the user provided Idle Instance CPU and/or memory percentage threshold then the Virtual Machine is recommended for deletion. If the usage is less than the user provided Underutilized Instance CPU and/or Memory percentage threshold then the Virtual Machine is recommended for downsizing. Both sets of Virtual Machines returned from this policy are emailed to the user.
+
+## Functional Details
+
+- The policy identifies all Virtual Machine resources using the bill data from Flexera API
+- The policy uses Azure API to get all utilization metrics for those resources during the lookback period.
+- The policy identifies all instances that have CPU and/or memory utilization below the user-specified idle thresholds and provides the relevant recommendation.
+- The recommendation provided for Idle Instances is a deletion action. The cluster or workspace can likely be deleted.
+- The recommendation provided for Underutilized Instances is a downsize action. The cluster or workspace can likely be downsized.
+
+### Policy savings details
+
+The policy includes the estimated monthly savings. The estimated monthly savings is recognized if the resource is deleted or downsized. Cost data from Flexera is used to retrieve and calculate the estimated savings.
+
+Because the virtual machine resources can often be short-lived and ephemeral, we use the actual cost during the lookback period to estimate Potential Monthly Savings.
+
+If the lookback period is 30 days (default value), the estimated savings for a Delete is 100% the actual cost during that 30 day period for a Downsize is 50% the actual cost during that 30 day period.
+
+If the lookback period is { 30 days (1 month), we calculate what % of 30 days the lookback period is and then use that ratio to estimate the savings for a full month period (30 days).  If the actual cost during a 15day lookback period was $100, the estimated savings calculation for a Delete is `(30/15) * $100`.  In this example, a Delete would estimate $200 Potential *Monthly* Savings and a Downsize would estimate $100 Potential *Monthly* Savings.
```

Comment:
"If the lookback period is { 30 days "

This { needs to be removed.

"15day"

Should be 15 day or 15-day

Reply:
Changes to `15-day` and `{` to `less than` in 851728c20c2a7cc5ca94dac013c36e98a3043cad